salt/_roster/kubernetes_nodes.py

```diff
@@ -0,0 +1,55 @@
+import fnmatch
+import logging
+log = logging.getLogger(__name__)
```

Comment:
Skip a line between the import and the constants declaration.

In fact I would even move the log declaration lower since here you're in the middle of the import statements.

Reply:
Done.